src/plugins/data_table/data_table.js

```diff
@@ -100,6 +100,26 @@ var LocalDataTable = (function() {
       return this.dataTable.fnSettings().fnRecordsTotal();
     },
 
+    hasUniqueValues: function() {
+      var hasUniques = false;
+
+      if (this.totalsRow && this.totalsRow.model) {
+        var totals = this.totalsRow.model.attributes;
+        var columns = this.totalsRow.model.keys();
+
+        _.each(columns, function(column) {
+          if (columns.indexOf(column + ".unique") !== -1) {
+            if (!(totals[column] instanceof Array) && totals[column] != totals[column + '.unique']) {
+              hasUniques = true;
+              return false;
+            }
+          }
+        });
+      }
```

Comment:
this method only acts on the `totalRows`.. so it should live down there (then you don't need the line later in this PR - `self.totalsRow.hasUniques = self.hasUniqueValues();` . 

Fortunately we have a base class that all Row classes derive from: https://github.com/Invoca/backdraft/blob/042fd8ea3f855a2aa8047b1310e53acdf518edaa/src/plugins/data_table/row.js#L9-L9